backend/src/v5/models/views.js

```diff
@@ -0,0 +1,35 @@
+/**
+ *  Copyright (C) 2021 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const Views = {};
+const db = require('../handler/db');
+const { stringToUUID } = require('../utils/helper/uuids');
+const { templates } = require('../utils/responseCodes');
+
+const colName = 'views';
+
+const getCollectionName = (model) => `${model}.${colName}`;
```

Comment:
Don't really need views to be a separate variable (v4 did it because of some inheritance faff - which we probably won't do in v5.)

```suggestion
const getCollectionName = (model) => `${model}.views`;
```